src/ClientData/CaptureDataTest.cpp

```diff
@@ -123,4 +132,47 @@ TEST_F(CaptureDataTest, UpdateScopeStatsIsCorrect) {
                    GetStats(kDurationsForSecondId, kSecondVariance));
 }
 
+TEST_F(CaptureDataTest, VarianceIsCorrectForLongDurations) {
+  for (TimerInfo timer : kTimerInfos) {
+    timer.set_end(timer.end() + kLargeInteger);
+    capture_data_.UpdateScopeStats(timer);
+  }
+
+  capture_data_.UpdateScopeStats(kTimerInfoWithInvalidScopeId);
+
+  ORBIT_LOG("VARIANCE %f", capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns());
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kFirstId).variance_ns(), kFirstVariance, 1.0);
+  EXPECT_NEAR(capture_data_.GetScopeStatsOrDefault(kSecondId).variance_ns(), kSecondVariance, 1.0);
+}
+
+TEST_F(CaptureDataTest, VarianceIsCorrectOnScimitarDataset) {
+  std::filesystem::path path = orbit_test::GetTestdataDir() / "scimitar_variance_and_durations.csv";
+  const ErrorMessageOr<std::string> file_content_or_error = orbit_base::ReadFileToString(path);
+  EXPECT_TRUE(file_content_or_error.has_value());
+  const std::string& file_content = file_content_or_error.value();
+  const std::vector<absl::string_view> tokens = absl::StrSplit(file_content, '\n');
+
+  double expected_variance;
+  EXPECT_TRUE(absl::SimpleAtod(*tokens.begin(), &expected_variance));
```

Comment:
So the first line is the expected variance? Maybe add a comment about the file here.